git-secrets

```diff
@@ -47,6 +47,10 @@ prepare_commit_msg_hook* prepare-commit-msg hook (internal only)"
 
 load_patterns() {
   git config --get-all secrets.patterns
+  local gitprohibited="$(git rev-parse --show-toplevel)/.gitprohibited"
+  if [ -e "$gitprohibited" ]; then
+    cat $gitprohibited | awk 'NF && $1!~/^#/'
```

Comment:
```suggestion
    awk 'NF && $1!~/^#/' "$gitprohibited"
```
`cat` is not necessary here, and the quotes are needed in case the path to the file has spaces